Requirements/Procedures/sp_BorrowedBooksReport.sql

```diff
@@ -0,0 +1,15 @@
+CREATE PROCEDURE sp_BorrowedBooksReport (
+ @StartDate Date,
+ @EndDate Date
+) AS
+BEGIN 
+ SELECT Books.Title , Loans.*,Borrowers.FirstName +' ' + Borrowers.LastName AS FullName
```

Comment:
we avoid using select * in stored procedures or views. if a column in the loans table is added or removed, any stored procedures or views relying on select * would break, causing errors in our project until they’re identified and fixed. instead, specify each required column explicitly to make the code resilient to such changes.